devops/scripts/control_tre.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+set -e
+
+if [[ "$1" == *"start"* ]]; then
+  echo -e "Starting Firewall - creating ip-config"
+  az network firewall ip-config create -f "fw-$TRE_ID" -g "rg-$TRE_ID" -n "AzureFirewallIpConfiguration0" --public-ip-address "pip-fw-$TRE_ID" --vnet-name "vnet-$TRE_ID" > /dev/null
```

Comment:
Should it check if it already has an IP config before creating one? What is the behavior if the firewall is already "started"?

Reply:
I'm just trying this, any idea how long it should take? Is there any output to let the user know something is happening or if any errors? I see the output is sent to `/dev/null`.

Reply:
It can take >5 mins to run - check you've rebuilt your container :)

When I was debugging locally I was frequently starting/stop - but I'll add an initial check to see if there is one there already

Reply:
Added check for az cli azure-firewall extension within check_dependencies.
Added additional status checks to reduce execution time